presto-testing/src/main/java/io/prestosql/testing/AbstractTestDistributedQueries.java

```diff
@@ -1060,9 +1060,9 @@ public void testViewColumnAccessControl()
                 privilege("orders", CREATE_VIEW_WITH_SELECT_COLUMNS));
 
         // verify selecting from a view over a table requires the view owner to have special view creation privileges for the table
-        assertAccessDenied(
+        assertWrappedAccessDenied(
```

Comment:
Why `assertAccessDenied` is not enough? Why do you care it is wrapped? To me it is implementation detail on which test should not rely on.

Reply:
`assertWrappedAccessDenied` checks the full message.
`assertAccessDenied` cannot be reused because it adds `".*Access Denied: "` at the beginning

Reply:
> assertAccessDenied cannot be reused because it adds ".*Access Denied: " at the beginning

And that is the point. `Access Denied: ` should be placed at the beginning

Reply:
done.